docs/docs/quickstart.md

```diff
@@ -7,7 +7,7 @@ slug: /quickstart
 Introduction
 ------------
 
-Welcome to the [Kurtosis][homepage] quickstart! This guide will take ~15 minutes and will walk you through building a basic Kurtosis package.
+Welcome to the [Kurtosis][homepage] quickstart! This guide will take ~15 minutes and will walk you through configuring and building a basic Kurtosis package that can be used as an environment for running multi-container tests.
```

Comment:
I would leave this section as-is, just to keep brevity

Reply:
```suggestion
Welcome to the [Kurtosis][homepage] quickstart! This guide will take ~15 minutes and will walk you through building a basic Kurtosis package.
```